pymetamap/SubprocessBackend.py

```diff
@@ -68,19 +68,19 @@ def extract_concepts(self, sentences=None, ids=None,
 
         input_file = None
         if sentences is not None:
-            input_file = tempfile.NamedTemporaryFile(delete=False)
+            input_file = tempfile.NamedTemporaryFile(mode="wb", delete=False)
         else:
-            input_file = open(filename, 'r')
-        output_file = tempfile.NamedTemporaryFile(delete=False)
+            input_file = open(filename, 'wb')
```

Comment:
Thanks! I will check out the changes and if everything works for me, I will merge the pull request. Thanks again for committing your changes.